flow/aggregates.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+from collections.abc import Iterable
+from hashlib import md5
+from hashlib import sha1
```

Comment:
`md5` is used to generate the id of an aggregate. No specific preference was there, it was just a suggestion that we could use `md5` for getting aggregate id.

Reply:
We use md5 for id calculation of Jobs.
I think that was the reason we were using md5 for calculating aggregate id.
@b-butler @vyasr Please suggest what should be done here.

Reply:
Is there a reason to use sha1 anywhere?

Reply:
Nope, no specific reason.
It was used while hashing `_JobOperation`, bundle id in `project.py`

Reply:
Okay, then I would be consistent here for this file as it is a bit cleaner.

Reply:
I think the intention here was to use `md5` everywhere rather than use `sha1` everywhere. We use `md5` for the `id` of jobs, so let's be consistent here. Additionally, `md5` is a faster hash function and it generates shorter hashes, which is good for keeping the directory names shorter for jobs and the strings shorter for aggregate stores. In general we're not concerned with the security implications; if that changes we'll need to change hashes everywhere in signac anyway (and we'd need to use something more secure than sha1).